scripts/bootstrap.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+err_report() {
+  echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
+
+IFS=$'\n\t'
+
+function grant_cluster_admin {
+  local username="$1"
+  kubectl create clusterrolebinding $username-cluster-admin-binding --clusterrole=cluster-admin --user=$username@gmail.com
+}
+
+function init_argocd {
+  kubectl create namespace argocd
+  kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+
+  # TODO(giri): Should be exposed through ingress controller, doing this temporarily
+  kubectl patch svc argocd-server -n argocd -p '{"spec": {"type": "LoadBalancer"}}'
```

Comment:
Have you considering the domain as well as tls cert?

Reply:
I'm deleting this because we don't always need to access ArgoCD service, registering app can happen through CRD and apply with usual kubectl

Reply:
fixed in https://github.com/cloudnative-id/kubeops/pull/8/commits/03070348dad660d19462738a35f638c2580ca76a